test_scripts/Polices/External_UCS/External_Consent_OFF/013_ATF_Policies_External_Consent_OFF_through_ignition_cycles_disallowed_by_external_consent_entities_omitted.lua

```diff
@@ -0,0 +1,350 @@
+--------------------------------------Requirement summary---------------------------------------------
+--[Policies] External UCS: "OFF" status between ignition cycles
+
```

Comment:
@istoimenova, Please put an additional req-t "[Policies] External UCS: "OFF" in case of omitted "disallowed_by_external_consent_entities" param in functional groupings"